mmv1/third_party/terraform/tests/resource_compute_instance_test.go

```diff
@@ -2646,6 +2671,16 @@ func testAccCheckComputeInstanceHasNetworkIP(instance *compute.Instance, network
 	}
 }
 
+func testAccCheckComputeInstanceHasNetworkPerformanceConfig(instance *computeBeta.Instance, bandwidthTier string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if instance.NetworkPerformanceConfig.TotalEgressBandwidthTier != bandwidthTier {
```

Comment:
Crash is thrown here because `instance.NetworkPerformanceConfig` is returning nil. I'll address why in my top-level review comment

Reply:
Ahh many thanks! I'll have a peek and see if I can address this before tomorrow AM.